wordpress-mobile/gutenberg-mobile
# Fix Web related breakage for using block wrappers

This PR's updates the GB reference to latest Master and brings fix that corrects the web breakage for blocks that use a block wrapper.

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/20772

To test:
 - Run the demo app
 - Check that no errors show up (red-screens) on start.

PR submission checklist:
